aws-backend/amplify/backend/function/fellowsEndpoint/src/app.js

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2017 - 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the License. A copy of the License is located at
+    http://aws.amazon.com/apache2.0/
+or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+*/
+
+
+
+const AWS = require('aws-sdk')
+var awsServerlessExpressMiddleware = require('aws-serverless-express/middleware')
```

Comment:
Is there a reason you used ES5 Var than const?

Reply:
ah I think it's the CLI tool using the old syntax! fixed :)